test/Microsoft.AspNetCore.Testing.Tests/OSSkipConditionAttributeTest.cs

```diff
@@ -92,5 +92,41 @@ public void Skips_WhenVersionsMatchesOutOfMultiple()
             // Assert
             Assert.False(osSkipAttribute.IsMet);
         }
+
+        [Fact]
```

Comment:
Could we add a couple of tests to https://github.com/aspnet/Testing/blob/dev/test/Microsoft.AspNetCore.Testing.Tests/OSSkipConditionTest.cs so we cover this E2E?